data/food/base/data.json

```diff
@@ -68,6 +68,41 @@
             "fdc_id": "168936",
             "alias": [],
             "comment": "all purpose flour, equivalent to type 550"
+        },
+        "food-ketchup": {
+            "name": "Ketchup",
+            "plural_name": "Ketchup",
+            "supermarket_category": "category-baking",
```

Comment:
Should this really be in baking? I would have looked for ketchup in condiments (with mustard, mayo, tabasco) or sauces (with pesto, bolognese, vanilla sauce).

Reply:
This is a good question. have you taken a look at the supermarket categories? How many categories would you think are worth adding to this base setup? https://github.com/TandoorRecipes/open-tandoor-data/blob/main/data/category/base/data.json

Reply:
See #8